psi4/driver/endorsed_plugins.py

```diff
@@ -51,3 +51,8 @@
         raise ImportError("""Psi4 plugin 'snsmp2' available, but scipy missing. Try `conda install scipy` or `pip install scipy`.""")
     else:
         pass
+
+try:
+    import resp
+except ImportError:
+    pass
```

Comment:
We may want a mechanism like the `XX_FOUND` CMake vars eventually, so we know which plugins are available in the script.  Still, that's another conversation for another day

Reply:
There's [this at runtime](https://github.com/psi4/psi4/blob/master/psi4/extras.py#L126-L152). I would agree with you above the whole `external/downstream/resp/CMakeLists.txt` treatment except that I realized it has a fatal flaw for pure-py plugins. Namely, it tries to import them to see if they're available, but that involves `import psi4`, but that's not built yet ....